wpilib/wpilib/i2c.py

```diff
@@ -128,7 +128,7 @@ def write(self, registerAddress, data):
             return True
         return False
 
-    def writeBulk(self, data):
+    def writeBulk(self, data: [bytes]):
```

Comment:
Is that right?

Reply:
Yeah, this is definitely wrong.

Reply:
Oh yeah this is what I was asking about in gitter. It should just be `bytes`?